service/frontend/admin_handler.go

```diff
@@ -2177,6 +2183,66 @@ func (adh *AdminHandler) getDLQWorkflowID(
 	)
 }
 
+func (adh *AdminHandler) GetDynamicConfigurations(
+	ctx context.Context,
+	req *adminservice.GetDynamicConfigurationsRequest,
+) (_ *adminservice.GetDynamicConfigurationsResponse, retError error) {
+	defer log.CapturePanic(adh.logger, &retError)
+	requestedKeys := req.GetDynamicConfigKeys()
+	dynamicConfig := make(map[string]*dc.ConstrainedValues, len(requestedKeys))
+
+	for _, key := range requestedKeys {
+		var dcEntry []dynamicconfig.ConstrainedValue
+		k := dynamicconfig.Key(key)
+		dcEntry = adh.dynamicClient.GetValue(k)
+
+		// get global default value
+		if dcEntry == nil {
+			defaultValue := dynamicconfig.GetDefaultValueForKey(k)
+			dcEntry = append(dcEntry, defaultValue)
+		}
+
+		protoValues := make([]*dc.ConstrainedValue, 0)
+		for _, ConstrainedValue := range dcEntry {
```

Comment:
use lowercase for variable names
```suggestion
		for _, cv := range dcEntry {
```